canonical-web-and-design/ubuntu.com
# Make the download server journey more of a wizard

## Done
Move the option step buttons to an equal level with the product offerings.

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/download/server
- Click through the steps and see they match [the copy doc](https://docs.google.com/document/d/12kr3JvN3ALMdr5IPOheMwOaf-ygvEulFo5D7x-0ag0k/edit#heading=h.go65ej6lc7f6)

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/3251
